docs/dev/build.md

```diff
@@ -344,56 +401,217 @@ There are two options how to use OpenVINO on Apple Silicon:
+
+1. Install dependencies:
+  ```bash
+  sudo apt-get update
+  sudo apt-get install -y git cmake scons build-essential
+  ```
+2. Clone the repositories:
+```
+git clone --recurse-submodules --single-branch --branch=master https://github.com/openvinotoolkit/openvino.git 
+git clone --recurse-submodules --single-branch --branch=master https://github.com/openvinotoolkit/openvino_contrib.git 
+```
+3. Go to the cloned `openvino` repository:
+
+  ```bash
+  cd openvino/
+  ```
+4. Create a build folder:
+
+  ```bash
+  mkdir build && cd build/
+  ```
+5. Build the OpenVINO Runtime:
+* for MYRIAD support only:
+  ```bash
+  cmake -DCMAKE_BUILD_TYPE=Release \
+        -DOPENVINO_EXTRA_MODULES=<OPENVINO_CONTRIB_PATH>/openvino_contrib/modules/arm_plugin \
+        -DARM_COMPUTE_SCONS_JOBS=$(nproc --all) \
+  .. && cmake --build . --parallel 
+  ```
+
+### Cross Compilation Using Docker*
+
+To cross-compile ARM CPU plugins using pre-configured `Dockerfile` you can use the following instruction: [Build OpenCV, OpenVINO™ and the plugin using pre-configured Dockerfile](https://github.com/openvinotoolkit/openvino_contrib/wiki/How-to-build-ARM-CPU-plugin#approach-1-build-opencv-openvino-and-the-plugin-using-pre-configured-dockerfile-cross-compiling-the-preferred-way).
+
+### Additional Build Options
+
+- To build Python API, install `libpython3-dev:armhf` and `python3-pip`
+  packages using `apt-get`; then install `numpy` and `cython` python modules
+  via `pip3`, adding the following options:
+   ```sh
+   -DENABLE_PYTHON=ON \
+   -DPYTHON_EXECUTABLE=/usr/bin/python3.7 \
+   -DPYTHON_LIBRARY=/usr/lib/arm-linux-gnueabihf/libpython3.7m.so \
+   -DPYTHON_INCLUDE_DIR=/usr/include/python3.7
+   ```
+
+</p>
 </details>
 
+<details><summary>WebAssembly</summary>
+<p>
+
+1. (Get sources) Clone submodules:
+```sh
+$ git clone https://github.com/openvinotoolkit/openvino.git
+$ cd openvino
+$ git submodule update --init
+```
+2. Run docker image and mount a volume with OpenVINO source code:
+```sh
+$ docker pull emscripten/emsdk
+$ docker run -it --rm -v `pwd`:/openvino emscripten/emsdk
+```
+3. (CMake configure) Run cmake configure step using helper emscripten command:
+```sh
+$ mkdir build && cd build
+$ emcmake cmake -DCMAKE_BUILD_TYPE=Release /openvino
+```
+4. (CMake build) Build OpenVINO project:
+```sh
+$ emmake make -j$(nproc)
+```
+`openvino.wasm` and `openvino.js` files are located in:
+- `<openvino_source_dir>/bin/ia32/Release/` on host machine file system.
+- `/openvino/bin/ia32/Release` in docker environment.
+These files can be used in browser applications. 
+
+</p>
+</details>
 
+<details><summary>Docker Image</summary>
 <p>
+
+For details on how to build Intel® Distribution of OpenVINO™ toolkit in a Docker image, follow this [guide](https://github.com/openvinotoolkit/docker_ci/tree/master/dockerfiles/ubuntu18/build_custom).
+
+</p>
 </details>
 
 ## Run application
```

Comment:
I think we can remove this section

Reply:
Also please add footer with links to other documents.

Reply:
Done in https://github.com/openvinotoolkit/openvino/pull/16063/commits/562c6c32912eb936d9ca384c7caa9cf772ce2fa2